e2e/jasmine.test.js

```diff
@@ -106,4 +106,14 @@ describe('examples/jasmine', () => {
       done()
     })
   }, 10000) // 10s timeout
+
+  describe('when test is pass but upload fails', () => {
+    test('it should not throw an error', done => {
+      exec("npm test spec/passed.spec.js", { cwd, env }, (error, stdout) => {
+        expect(error).toBeNull()
```

Comment:
It isn't immediately obvious to me why this imply a `upload fails`, but I guess this is hitting our production API and we expect a 4xx error?

Reply:
Ah yes, it's not explicit, but the test will make a call to production API, that will return `401`. I'll see if I can make it explicit.